docs/sources/developers/http_api/access_control.md

```diff
@@ -527,11 +527,60 @@ Content-Type: application/json; charset=UTF-8
 | 403  | Access denied.                                                       |
 | 500  | Unexpected error. Refer to body and/or server logs for more details. |
 
+### List your permissions
+
+`GET /api/access-control/users/permissions`
+
+Lists the permissions granted to the signed in user.
+
+#### Required permissions
+
+No permission is required.
+
+#### Query parameters
+
+| Param       | Type    | Required | Description                                                           |
+| ----------- | ------- | -------- | --------------------------------------------------------------------- |
+| reloadcache | boolean | No       | A flag to get permissions from store and reload the permission cache. |
```

Comment:
nit: is `store` a widely used term? I think `data store` or `database` would be easier to understand.